app/scripts/controller-init/messengers/assets/token-rates-controller-messenger.ts

```diff
@@ -0,0 +1,61 @@
+import { Messenger } from '@metamask/base-controller';
+import {
+  NetworkControllerGetStateAction,
+  NetworkControllerGetNetworkClientByIdAction,
+  NetworkControllerStateChangeEvent,
+} from '@metamask/network-controller';
+import {
+  AccountsControllerGetSelectedAccountAction,
+  AccountsControllerGetAccountAction,
+  AccountsControllerSelectedEvmAccountChangeEvent,
+} from '@metamask/accounts-controller';
+import { PreferencesControllerStateChangeEvent } from '@metamask/preferences-controller';
+import {
+  TokensControllerGetStateAction,
+  TokensControllerStateChangeEvent,
+} from '@metamask/assets-controllers';
+
+type Actions =
```

Comment:
I need to double check the docs, but if our controllers export the `AllowedActions` and `AllowedEvents` why can't we reuse those.